src/Shotr.Ui/Forms/Settings/SettingsForm.cs

```diff
@@ -13,30 +13,6 @@ namespace Shotr.Ui.Forms.Settings
     public partial class SettingsForm : DpiScaledForm
     {
         private readonly BaseSettings _settings;
-
-        /*
```

Comment:
Didn't notice how hardcoded this previously was. Glad you were able to refactor this.